design/prototype-version-7/task-list/README.md

```diff
@@ -0,0 +1,140 @@
+
+<p>The second row has a link to “Add and edit your questions“ and a grey highlighted tag on the far right that reads “not started“.</p>
+
+<p>The third row has a link to “Review summary page and add declaration“ and a grey highlighted tag on the far right that reads “not started“.</p>
+
+<p>The fourth row has a link to “Add information about what happens next“ and a grey highlighted tag on the far right that reads “not started“.</p>
+</ul>
+
+<br>  
+
+Step “2. Set form responses“ which has 2 rows under it.
+
+<ul> 
+<p>The first row has a link to “Set the email address completed forms will be sent to“ and a grey highlighted tag on the far right of the row that reads “not started“.</p>
+
+<p>The second row has an inactive link to “Enter the email address confirmation code“ and a grey highlighted tag on the far right of the row that reads “Cannot start yet“. This inactive link changed to an active when the form creator completes the previous step and the grey highlighted status tag would change to “not started“.</p>
+</ul>
+
+<br>  
+
+Step “3. Get your form ready to go live“ which has 2 rows under it.
+
+<ul> 
+<p>The first row has a link to “Provide link to your privacy information“ and a grey highlighted tag on the far right of the row that reads “not started“.</p>
+
+<p>The second row has a link to “Provide link to your accessibility statement“ and a grey highlighted tag on the far right of the row that reads “not started“.</p>
+</ul>
+
+<br>  
+
+Step “4. Publish your form“ which has 2 rows under it.
+
+<ul>
+<p>The first row has a link to “Make your form live“ and a grey highlighted tag on the far right of the row that reads “not started“.</p>
+
+<p>The second row has a link to “How to publish the form on GOV.UK“ and a grey highlighted tag on the far right of the row that reads “not started“.</p>
+</ul>
+
+<br>  
+
+## What we saw
+
+- Users found the task list clear, easy to use and logical to navigate
+- Users found the progress labels helpful to work through the list and used them regularly.  
+- All tasks were successfully completed apart from email setup which was not entirely clear to one low digital confidence user. 
+- All users completed the tasks in order, but when questioned said they understood it could be completed in any order.
+- The link text to “Review summary page and add declaration“ was not clear to all users. This became clearer for some when they clicked the link to get to the ‘what happens next’ page.
+- When users clicked on the link to “Add and edit your questions“ tasks, and added their questions, the grey highlighted tag on the far right of the row changed to a light blue tag that reads “in progress“ as shown below. Users were unsure how to move the tag to “complete“
+
+<br>
+
+![Step “1. Create your form” tasks. Screenshot](../screenshots/003-1-Task-list-page-tags-focus-create-your-form.png)
```

Comment:
```suggestion
![Step “1. Create your form” tasks. Screenshot](../../prototype-version-6/screenshots/003-1-Task-list-page-tags-focus-create-your-form.png)
```
I will make sure to add this screenshot into V6 or I will update the link when writing up V6